docs/pages/rows/row_dragging.py

```diff
@@ -12,42 +12,319 @@
+> ColDef it will take precedence over this, except when `rowDragEntireRow=true`.
+
+The example below shows dragging with custom text. The following can be noted:
+
+- When you drag a row, the `rowDragText` callback will add the host city depending on the year to the floating drag
+  element, like "(London Olympics)" if year of the dragged row is 2012
+
+Here is the function added to the dashAgGridFunctions.js file in the assets folder:
+
+```js
+const hostCities = {2000: "Sydney", 2004: "Athens", 2008: "Beijing", 2012: "London",}
+
+dagfuncs.rowDragText = function (params) {
+    const {year} = params.rowNode.data;
+    if (year in hostCities) {
+        return `${params.defaultTextValue} (${hostCities[year]} Olympics)`
+    }
+    return params.defaultTextValue;
+}
+```
+"""
+
+text4 = """
+### Custom Row Drag Text with Multiple Draggers
+
+If the grid has more than one column set with `rowDrag=true`, the `rowDragText` callback can be set in the `colDef`.
+
+```python
+columnDefs = [
+    {'field': 'athlete', 'rowDrag': True, "rowDragText": {"function": "athleteRowDragText(params)"}},
+    {'field': 'country', 'rowDrag': True},
+]
+```
+
+The example below shows dragging with custom text and multiple column draggers. The following can be noted:
+
+- When you drag a row by the **country** row dragger, the `rowDragText` callback will add the host city depending on the
+  year to the floating drag element, like "(London Olympics)" if year of the dragged row is 2012
+- When you drag the row by the **athlete** row dragger, the `rowDragText` callback in the grid options will be
+  overridden by the one in the `colDef` and will display the number of athletes selected.
+
+Here are the functions added to the dashAgGridFunctions.js file in the assets folder:
+
+```js
+const hostCities = {2000: "Sydney", 2004: "Athens", 2008: "Beijing", 2012: "London",}
+
+dagfuncs.rowDragText = function (params) {
+    const {year} = params.rowNode.data;
+    if (year in hostCities) {
+        return `${params.defaultTextValue} (${hostCities[year]} Olympics)`
+    }
+    return params.defaultTextValue;
+}
+
+dagfuncs.athleteRowDragText = function (params) {
+    return `${params.rowNodes.length} athlete(s) selected`
+}
+"""
+
+text5 = """
+### Row Dragger inside Custom Cell Renderers
+
+Due to the complexity of some applications, it could be handy to render the Row Drag Component inside of a Custom Cell
+Renderer. This can be achieved, by using the `registerRowDragger` method in
+the [ICellRendererParams](https://www.ag-grid.com/react-data-grid/component-cell-renderer/?#reference-ICellRendererParams-registerRowDragger)
+as follows:
+
+```js
+// your custom cell renderer code
+
+// this will hold the reference to the element you want to act as row dragger.
+myRef = React.createRef();
+
+componentDidMount()
+{
+    this.props.registerRowDragger(this.myRef.current);
+}
+```
```

Comment:
Perhaps remove this snippet since we can't  use `componentDidMount()`

Reply:
I wasn't sure about this one, I thought maybe it is possible to create custom cell renderer with stateful component, but that's right, I didn't see any example like that (maybe I will try it later if it is possible 😃 )
I modified this snippet with the syntax of the example